agave_app/tfeditor/gradients.cpp

```diff
@@ -132,6 +151,79 @@ GradientEditor::GradientEditor(const Histogram& histogram, QWidget* parent)
   vbox->addWidget(m_customPlot);
 }
 
+void
+GradientEditor::changeEvent(QEvent* event)

```

Comment:
everything inside here is for events that change dark to light mode.  it is also called when the widget is first shown.